### ISSUE TITLE
Uncaught Error: Basic authentication requires both a username and password to be set

### ISSUE THREAD

**Issue by gramimoss** [jonasws/github-utils#1]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.7
**System**: Linux Mint
**Thrown From**: [github-utils](https://github.com/jonasws/github-utils) package, v0.6.1

### Stack Trace

Uncaught Error: Basic authentication requires both a username and password to be set

```
At /home/graeme/.atom/packages/github-utils/node_modules/github/index.js:425

Error: Basic authentication requires both a username and password to be set
  at [object Object].authenticate (/home/graeme/.atom/packages/github-utils/node_modules/github/index.js:425:19)
  at authenticate (/home/graeme/.atom/packages/github-utils/lib/github-utils.js:46:10)
  at atom-workspace.viewPullRequests (/home/graeme/.atom/packages/github-utils/lib/github-utils.js:71:3)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/usr/share/atom/resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -2:31.7.0 git-plus:init (atom-pane.pane.active)
     -2:24.7.0 command-palette:toggle (atom-pane.pane.active)
     -2:19.6.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -2:19.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:19.2.0 git-plus:checkout-remote (atom-pane.pane.active)
     -0:50.1.0 command-palette:toggle (atom-pane.pane.active)
  5x -0:40.4.0 core:move-down (atom-text-editor.editor.mini.is-focused)
  3x -0:35.7.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -0:33.4.0 core:move-right (atom-text-editor.editor.mini.is-focused)
  2x -0:31 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:28.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:28.4.0 git-plus:checkout (atom-pane.pane.active)
     -0:20.3.0 command-palette:toggle (atom-pane.pane.active)
  3x -0:07.9.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:00.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.9.0 github-utils:view-pull-request (atom-pane.pane.active)
```

### Config

```json
{
  "github-utils": {}
}
```

### Installed Packages

```coffee
# User
git-plus, v5.4.7
github-utils, v0.6.1
remote-sync, v3.1.2

# Dev
No dev packages
```

**Comment by jonasws** [comment]
From your config you have not set a username and authentication token in the settings view. I should indeed add some more error handling, though this is indeed the expected behavior,

**Comment by jonasws** [comment]
Fixed in version [0.6.2](https://github.com/jonasws/github-utils/commit/c57c0e16c97761b723cc38551d469d517cdc8d6d).
